exercises/rna-transcription/README.md

```diff
@@ -0,0 +1,41 @@
+# RNA Transcription
```

Comment:
you are right that this should be RNA transcription, though the generator says "Rna Transcription" instead. That's a bit annoying discrepancy, would like to figure out how to change that so that the generator doesn't try to change it back when run on this track.

No change needs to happen for this PR, but I need to note it down for later use.

Reply:
I noted it at https://github.com/exercism/configlet/issues/76